lintro/ai/interactive.py

```
@@ -0,0 +1,412 @@
+            else:
+                choice = choice.lower()
+
+            if choice == KEY_SHOW_DIFF:
+                _show_group_diffs(console, fixes)
+                console.print()
+                continue
+            if choice == KEY_TOGGLE_VALIDATE:
+                validate_mode = not validate_mode
+                state = "enabled" if validate_mode else "disabled"
+                console.print(
+                    f"  [dim]Per-group validation {state} " "(no fixes applied).[/dim]",
```

Ruff (0.15.6)

[warning] Implicitly concatenated string literals on one line

(ISC001)